### PR TITLE
Insurance data set

### PR DESCRIPTION
**Pull request by jjbrosnan**
Contains readme and updates to base readme

### CODE REVIEW

File: Insurance/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+## Fields in the file
+
+`Age` - Age of the patient in years
+`Sex` - Biological sex of the patient
+`BMI` - The patient's BMI
+`Children` - The number of children the patient has
+`Smoker` - `Yes` or `No` if the patient smokes cigarettes or not
+`Region` - The region of the United States the patient resides in
+`Expenses` - The patient's insurance expenses# Insurance dataset
```
Comment (margaretkennedy) [inline Insurance/README.md:13]:
```suggestion
`Expenses` - The patient's insurance expenses
```

File: Insurance/README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+This folder contains one sub-folder, `csv`, which contains the `insurance.csv` file.  This file contains health indicators of patients and their insurance charges.  These can be used to predict insurance charges given their own health indicators.
```
Comment (margaretkennedy) [inline Insurance/README.md:15]: accidentally duplicated?

**Review by margaretkennedy — CHANGES_REQUESTED** [review]

File: Insurance/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# Insurance dataset
+
+This folder contains one sub-folder, `csv`, which contains the `insurance.csv` file.  This file contains health indicators of patients and their insurance charges.  These can be used to predict insurance charges given their own health indicators.
```
Comment (chipkent) [inline Insurance/README.md:3]:
```suggestion
This folder contains one sub-folder, `csv`, which contains the `insurance.csv` file.  This file contains health indicators of patients and their insurance charges.  These can be used to predict insurance charges based upon health indicators.
```

File: Insurance/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+`Sex` - Biological sex of the patient
+`BMI` - The patient's BMI
+`Children` - The number of children the patient has
+`Smoker` - `Yes` or `No` if the patient smokes cigarettes or not
+`Region` - The region of the United States the patient resides in
+`Expenses` - The patient's insurance expenses# Insurance dataset
+
+## Source and license
+
+This data was built from [Kaggle](https://kaggle.com).  The original link, authored by Brandon Yong, can be found [here](https://www.kaggle.com/brandonyongys/insurance-charges/notebook#Basic-data-information).  It is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.
```
Comment (chipkent) [inline Insurance/README.md:17]:
```suggestion
This data was obtained from [Kaggle](https://kaggle.com).  The original analysis and data, authored by Brandon Yong, can be found [here](https://www.kaggle.com/brandonyongys/insurance-charges/notebook#Basic-data-information).  This data is provided here for demonstrative use without any warranty as to the accuracy, reliability, or completeness of the data.
```

**Review by chipkent — CHANGES_REQUESTED** [review]

**Pull request merged**